recipe/meta.yaml

```diff
@@ -17,14 +17,14 @@ build:
 
 requirements:
   build:
-    - python >= 3.5
+    - python >=3.5
```

Comment:
Is that a bug in the conda forge tools? I had thought a space worked fine in conda build.